client/src/components/selects/stringMultiSelect.tsx

```diff
@@ -0,0 +1,58 @@
+import { MenuItem } from '@blueprintjs/core';
+import { ItemPredicate, MultiSelect } from '@blueprintjs/select';
+import React from 'react';
+
+interface Props {
+  items: string[];
+  selectedItems: string[];
+  onItemSelect: (item: string) => void;
+  onRemoveTag?: (item: string, ind: number) => void;
+}
+
+const StringMultiSelect = ({
```

Comment:
Where is this used?

Reply:
where the OwnerSelect should be (2 places: newSource and database drawer)